spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/stores/ISyncableStore.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2015-2017 Boundless, http://boundlessgeo.com
```

Comment:
FYI...You only need to put 2017 if the file was created this year. If you're updating a file, you can update the header to include the year if it's not there.